geonotebook/layers.py

```diff
@@ -21,25 +105,35 @@ class GeonotebookLayer(object):
     # some kind of functionality (e.g. the annotatoin layer).
     _expose_as = None
 
-    _type = None
-
-    def __init__(self, name, remote, **kwargs):
+    def __init__(self, name, remote, data, **kwargs):
```

Comment:
Now I see some more code, I am wondering if we could make the signature

def __init__(self, name, remote, vis_option) and have users pass the vis_option (raster style , vis object) to be passed to it?